### ISSUE TITLE
[xas_viewer] unify Wx data importers

### ISSUE THREAD

**Issue by maurov** [xraypy/xraylarch#367]
Currently there are three separate Wx data importer frames (in `larch.wxlib`): 1) columnframe, 2) specfile_importer, 3) athenaproject_importer. It is difficult to  maintain the three of them as new features are added. It would result beneficial to have only one Wx frame capable of importing all types of data supported in Larch.

**Comment by newville** [comment]
@maurov yes, I agree these should be using a more common interface especially for the part that converts the 2d data array read in from the source to energy/mu arrays.
